lib/src/slider/Slider.stories.tsx

```diff
@@ -0,0 +1,172 @@
+          <DxcSlider
+            label="Slider"
+            helperText="Help message"
+            disabled
+            value={40}
+            minValue={0}
+            maxValue={50}
+            showLimitsValues
+            showInput
+            marks
+            step={5}
+          />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Continuous slider" theme="dark" level={4} />
+          <DxcSlider value={65} label="Slider" helperText="Help message" showLimitsValues />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Discrete slider" theme="dark" level={4} />
+          <DxcSlider value={20} label="Slider" helperText="Help message" showLimitsValues marks step={5} />
+        </ExampleContainer>
+        <ExampleContainer>
+          <Title title="Discrete slider with input" theme="dark" level={4} />
+          <DxcSlider
+            value={20}
+            minValue={0}
+            maxValue={50}
+            label="Slider"
+            helperText="Help message"
+            showLimitsValues
+            showInput
+            marks
+            step={10}
+          />
+        </ExampleContainer>
+      </DarkContainer>
+    </BackgroundColorProvider>
+    <Title title="Margins" theme="light" level={2} />
+    <ExampleContainer>
+      <Title title="Xxsmall" theme="light" level={4} />
+      <DxcSlider label="Xxsmall" margin="xxsmall" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xsmall" theme="light" level={4} />
+      <DxcSlider label="Xsmall" margin="xsmall" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Small" theme="light" level={4} />
+      <DxcSlider label="Small" margin="small" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Medium" theme="light" level={4} />
+      <DxcSlider label="Medium" margin="medium" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Large" theme="light" level={4} />
+      <DxcSlider label="Large" margin="large" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xlarge" theme="light" level={4} />
+      <DxcSlider label="Xlarge" margin="xlarge" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcSlider label="Xxlarge" margin="xxlarge" />
+    </ExampleContainer>
+    <Title title="Sizes" theme="light" level={2} />
+    <ExampleContainer>
+      <Title title="Large" theme="light" level={4} />
+      <DxcSlider label="Large" size="medium" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xlarge" theme="light" level={4} />
+      <DxcSlider label="Xlarge" size="large" />
+    </ExampleContainer>
+    <ExampleContainer>
+      <Title title="Xxlarge" theme="light" level={4} />
+      <DxcSlider label="Xxlarge" size="fillParent" />
+    </ExampleContainer>
```

Comment:
Title and slider label here should be `FillParent`.

Reply:
Done!